game/scripts/npc/abilities/spectre_dispersion.txt

```diff
@@ -38,13 +38,13 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "shard_dagger_damage"                             "400" //OAA
+        "shard_dagger_damage"                             "300"
```

Comment:
🤔

Reply:
Fuck spectre honestly shard_dagger_damage is too high to begin with, Darkonius was pretty stupid to make a change like this to an already imba hero

Reply:
This is the damage threshold when spectral dagger procs, so the lower the value the better. Bad name for kv so it's confusing.